src/material/list/testing/list-harness-filters.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {BaseHarnessFilters} from '@angular/cdk/testing';
+
+export interface ListHarnessFilters extends BaseHarnessFilters {}
+
+export interface ActionListHarnessFilters extends BaseHarnessFilters {}
+
+export interface NavListHarnessFilters extends BaseHarnessFilters {}
+
+export interface SelectionListHarnessFilters extends BaseHarnessFilters {}
+
+export interface BaseListItemHarnessFilters extends BaseHarnessFilters {
+  text?: string | RegExp;
```

Comment:
Would it make sense to have `subheader` as a filter here?

Reply:
I don't think so. Because the list doesn't have hierarchical sections, its not always clear if an item belongs to a header or not. e.g.:

```
Subheader 1
  Item 1
  Item 2
-----------
  Item 3 <-- under Subheader 1?
```